src/util/os.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"runtime"
+)
+
+type osName string
+
+const OS = osName(runtime.GOOS)
+
+// Returns one of the arguments, based on operating system
+func (os osName) Sieve(onUnix interface{}, onWindows interface{}) interface{} {
```

Comment:
This file can be removed too. Once we modify the command in reader_test.go, this function is only needed in terminal_test.go. No need to extend the public API.

Reply:
I have addressed the issues in the commit below. But I removed the Sieve method, it felt wrong to keep it with only one occurrence.